dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -93,13 +94,27 @@ export const Ad = ({
 	contentType,
 	commercialProperties,
 	rtcConfig,
+	adTargeting,
 }: AdProps) => {
 	const adSizes = isSticky ? stickySizes : inlineSizes;
 	// Set Primary ad size as first element (should be the largest)
 	const [{ width, height }] = adSizes;
 	// Secondary ad sizes
 	const multiSizes = adSizes.map((e) => `${e.width}x${e.height}`).join(',');
 
+	const adTargetingMapped: AdTargetParam[] = [];
+	if (adTargeting.disableAds !== true) {
+		adTargetingMapped.push({
+			name: 'sens',
+			value: String(adTargeting.customParams.sens),
+		});
+
+		adTargetingMapped.push({
+			name: 'urlkw',
+			value: adTargeting.customParams.urlkw as string[],
+		});
+	}
```

Comment:
Optional suggestion, but since this code doesn't have any side effects it might be nice to move this to its own function outside the component, so that below on L137 you can just call the function like

```typescript
...mapAdTargeting(adTargeting)
```

Reply:
Yeah this is better!